pkg/reconciler/dependencybuild/dependencybuild_test.go

```diff
@@ -249,7 +251,7 @@ func TestStateBuilding(t *testing.T) {
 		db.Namespace = metav1.NamespaceDefault
 		db.Name = "test"
 		db.Status.State = v1alpha1.DependencyBuildStateBuilding
-		db.Status.CurrentBuildRecipe = &v1alpha1.BuildRecipe{Image: "quay.io/sdouglas/hacbs-jdk11-builder:latest"}
+		db.Status.CurrentBuildRecipe = &v1alpha1.BuildRecipe{Image: "quay.io/redhat-appstudio/hacbs-jdk11-builder:latest"}
```

Comment:
should we not be using the `JVM_BUILD_SERVICE_JDK11_BUILDER_IMAGE` env var here, with perhaps a sha specific version of `quay.io/redhat-appstudio/hacbs-jdk11-builder` as a default?

Reply:
This is a unit test, it does not actually use the image but just verifies that it uses the expected one. This could be any string and it would be ok.

Reply:
duh right